stories/TextArea/TextArea.story.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { Header } from '../shared';
+import TextArea from '../../components/TextArea';
+import { TextAreaComponent } from '../../components/TextArea/TextArea';
+import { Default, Disabled, Required, WithAutoSize, WithCustomAutoSize, WithHelperText, WithError, WithLabel } from './TextArea.stories.jsx';
+
+<Meta title="Forms/TextArea" component={TextArea}  />
```

Comment:
Definition for component is wrong here. On the canvas tab haven't correct controls.